### ISSUE TITLE
velero backup describe doesn't work 

### ISSUE THREAD

**Issue by ruzickap** [vmware-tanzu/velero#3682]
**What steps did you take and what happened:**

I installed Velero 1.6.0 to store backups in S3 using helm chart:

```bash
helm install --version 2.17.1 --namespace velero --create-namespace --values - velero vmware-tanzu/velero << EOF
initContainers:
  - name: velero-plugin-for-aws
    image: velero/velero-plugin-for-aws:v1.2.0
    imagePullPolicy: Always
    volumeMounts:
      - mountPath: /target
        name: plugins
  - name: velero-plugin-for-csi
    image: velero/velero-plugin-for-csi:v0.1.2
    imagePullPolicy: Always
    volumeMounts:
      - mountPath: /target
        name: plugins
metrics:
  serviceMonitor:
    enabled: true
configuration:
  provider: aws
  backupStorageLocation:
    bucket: ${CLUSTER_FQDN}
    prefix: velero
    config:
      region: ${AWS_DEFAULT_REGION}
  volumeSnapshotLocation:
    name: aws
    config:
      region: ${AWS_DEFAULT_REGION}
  features: EnableCSI
credentials:
  secretContents:
    cloud: |
      [default]
      aws_access_key_id=${AWS_ACCESS_KEY_ID}
      aws_secret_access_key=${AWS_SECRET_ACCESS_KEY}
deployRestic: true
EOF
```

Then I did the backup of my namespaces which was successful: 

```bash
velero backup create backup-vault --ttl 24h --include-namespaces=vault --wait
```

Then I used the `velero backup describe` to see the details about backup - worked fine:

```bash
velero backup describe backup-vault --features=EnableCSI

Name:         backup-vault
Namespace:    velero
Labels:       velero.io/storage-location=default
Annotations:  velero.io/source-cluster-k8s-gitversion=v1.19.6-eks-49a6c0
              velero.io/source-cluster-k8s-major-version=1
              velero.io/source-cluster-k8s-minor-version=19+

Phase:  Completed

Errors:    0
Warnings:  0

Namespaces:
  Included:  vault
  Excluded:  <none>

Resources:
  Included:        *
  Excluded:        <none>
  Cluster-scoped:  auto

Label selector:  <none>

Storage Location:  default

Velero-Native Snapshot PVs:  auto

TTL:  24h0m0s

Hooks:  <none>

Backup Format Version:  1.1.0

Started:    2021-04-15 19:24:28 +0200 CEST
Completed:  2021-04-15 19:24:44 +0200 CEST

Expiration:  2021-04-16 19:24:21 +0200 CEST

Total items to be backed up:  63
Items backed up:              63

Velero-Native Snapshots: <none included>

CSI Volume Snapshots:  1 included (specify --details for more information)
```

If I tried to use `velero backup describe --details` - the command hanged without any output:

```bash
velero backup describe backup-vault --details --features=EnableCSI
```

In the Velero logs I can see:

```bash
❯ k logs -n velero velero-7657d5b8d4-hmgl5 --tail=10
time="2021-04-15T17:37:03Z" level=error msg="Error updating download request" controller=download-request downloadRequest=velero/backup-vault-20210415193652 error="downloadrequests.velero.io \"backup-vault-20210415193652\" not found" logSource="pkg/controller/download_request_controller.go:81"
time="2021-04-15T17:37:12Z" level=error msg="Error updating download request" controller=download-request downloadRequest=velero/backup-vault-20210415193549 error="downloadrequests.velero.io \"backup-vault-20210415193549\" not found" logSource="pkg/controller/download_request_controller.go:81"
time="2021-04-15T17:37:13Z" level=error msg="Error updating download request" controller=download-request downloadRequest=velero/backup-vault-20210415193652 error="downloadrequests.velero.io \"backup-vault-20210415193652\" not found" logSource="pkg/controller/download_request_controller.go:81"
time="2021-04-15T17:37:23Z" level=info msg="Validating backup storage location" backup-storage-location=default controller=backup-storage-location logSource="pkg/controller/backup_storage_location_controller.go:114"
time="2021-04-15T17:37:23Z" level=info msg="Backup storage location valid, marking as available" backup-storage-location=default controller=backup-storage-location logSource="pkg/controller/backup_storage_location_controller.go:121"
time="2021-04-15T17:37:33Z" level=error msg="Error updating download request" controller=download-request downloadRequest=velero/backup-vault-20210415193652 error="downloadrequests.velero.io \"backup-vault-20210415193652\" not found" logSource="pkg/controller/download_request_controller.go:81"
time="2021-04-15T17:37:41Z" level=error msg="Error updating download request" controller=download-request downloadRequest=velero/backup-vault-20210415192643 error="downloadrequests.velero.io \"backup-vault-20210415192643\" not found" logSource="pkg/controller/download_request_controller.go:81"
time="2021-04-15T17:38:01Z" level=error msg="Error updating download request" controller=download-request downloadRequest=velero/backup-vault-20210415192703 error="downloadrequests.velero.io \"backup-vault-20210415192703\" not found" logSource="pkg/controller/download_request_controller.go:81"
time="2021-04-15T17:38:09Z" level=error msg="Error updating download request" controller=download-request downloadRequest=velero/backup-vault-20210415192711 error="downloadrequests.velero.io \"backup-vault-20210415192711\" not found" logSource="pkg/controller/download_request_controller.go:81"
time="2021-04-15T17:38:14Z" level=error msg="Error updating download request" controller=download-request downloadRequest=velero/backup-vault-20210415193652 error="downloadrequests.velero.io \"backup-vault-20210415193652\" not found" logSource="pkg/controller/download_request_controller.go:81"
```

`velero backup logs backup-vault` didn't work as well...

**What did you expect to happen:**

`velero backup describe --details` or `velero backup logs` should give me proper output

**Environment:**

- Velero version (use `velero version`): `v1.6.0`
- Velero features (use `velero client config get features`): `<NOT SET>`
- Kubernetes version (use `kubectl version`): `v1.21.0`
- Kubernetes installer & version: Amazon EKS 1.19
- Cloud provider or hardware configuration: AWS
- OS (e.g. from `/etc/os-release`): AmazonLinux2

**Comment by ajchiarello** [comment]
Seeing the same issues with kubernetes v1.19.9. I can access the details by downgrading velero below 1.6.0, but the download request hangs when using the 1.6.0 chart.

**Comment by alxndr13** [comment]
Same issue here. 
- Kubernetes Version: 1.17.12
- Kubernetes Installer: Amazon EKS 1.17
- velero version 1.6.0 
- Helm Chart Version: velero-2.17.1

Trying to restore from a non-EKS Cluster with version 1.17.7. Backend is S3.

**Comment by zubron** [comment]
Hi @ruzickap - Thanks for reporting this. I think this may have been due to an issue with the initial 1.6.0 chart which did not include the updated CRDs. This was fixed with vmware-tanzu/helm-charts#248 and the new chart version is v2.17.2. Given that you have an existing installation, and I don't think helm upgrade supports upgrading CRDs, you will need to upgrade the CRDs using the [1.6.0 upgrade instructions](https://velero.io/docs/v1.6/upgrade-to-1.6/).

**Comment by ruzickap** [comment]
Thank you.

I was able to make it running when using the Helm Chart `2.17.2`.

Closing...

**Comment by laimison** [comment]
> Hi @ruzickap - Thanks for reporting this. I think this may have been due to an issue with the initial 1.6.0 chart which did not include the updated CRDs. This was fixed with [vmware-tanzu/helm-charts#248](https://github.com/vmware-tanzu/helm-charts/pull/248) and the new chart version is v2.17.2. Given that you have an existing installation, and I don't think helm upgrade supports upgrading CRDs, you will need to upgrade the CRDs using the [1.6.0 upgrade instructions](https://velero.io/docs/v1.6/upgrade-to-1.6/).

Thanks, this method to update CRDs helped to solve same issue on Kubernetes cluster built with Kubespray too.
